ably/rest/push.py

```diff
@@ -0,0 +1,37 @@
+
+class Push(object):
+
+    def __init__(self, ably):
+        self.__ably = ably
+        self.__admin = PushAdmin(ably)
+
+    @property
+    def admin(self):
+        return self.__admin
+
+
+class PushAdmin(object):
+
+    def __init__(self, ably):
+        self.__ably = ably
+
+    @property
+    def ably(self):
+        return self.__ably
+
+    def publish(self, recipient, data):
```

Comment:
I'm comparing with the analogous RET publish method (https://github.com/ably/ably-python/blob/master/ably/rest/channel.py#L70)

There, we have inline doc strings - should we also do that here?